packages/template-retail-react-app/app/hooks/use-datacloud.js

```diff
@@ -38,13 +32,20 @@ export class DataCloudApi {
             guestId: args.guestId,
             siteId: this.siteId,
             sessionId: args.sessionId,
-            deviceId: args.deviceId,
+            deviceId: args.customerId || args.guestId,
```

Comment:
I'm not familiar with the data cloud integration. What is this device ID and why are we setting it to the customer / guest id?